### PR TITLE
LQR based altitude control

### PR DESCRIPTION
**Pull request by gunturiCM**
# This is a draft.
## Main Changes
This is an attempt to implement an LQR-based altitude control algorithm, which runs on Raspberry Pi.

### CODE REVIEW

**Comment by alphaville** [comment]
@gunturiCM This is getting too lengthy and it will be difficult to review. Let us set a realistic goal that will allow us to merge it into `dev` soon. When you're ready please invite @boyle26 too.

File: raspberry/read_sbus/__pycache__/radioDataParser.cpython-37.pyc
Comment (alphaville) [inline raspberry/read_sbus/__pycache__/radioDataParser.cpython-37.pyc:1]: Let us include `.pyc` in our `.gitignore`

File: raspberry/sensors/camera_sensor.py
Comment (alphaville) [inline raspberry/sensors/camera_sensor.py:1]: PRs must be atomic

File: raspberry/sensors/pressure_sensor.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
+# TODO: implement proper code for a particular sensor
+# TODO: add proper Doc strings
+# NOTE: for now, we are just passing dummy values
+
+from filters import median_filter
```
Comment (alphaville) [inline raspberry/sensors/pressure_sensor.py:5]: From PEP8: Imports should be grouped in the following order:

- Standard library imports.
- Related third party imports.
- Local application/library specific imports

Study [this](https://peps.python.org/pep-0008/#imports)

**Comment by gunturiCM** [comment]
Closing because a working altitude hold control is implemented in #107.

**Pull request closed without merging**